packages/plugin-ext/compile.tsconfig.json

```diff
@@ -73,6 +73,9 @@
     },
     {
       "path": "../workspace/compile.tsconfig.json"
+    },
+    {
+      "path": "../console/compile.tsconfig.json"
```

Comment:
minor: do you mind updating the location so it is in alphabetical order like the rest?